modules/rubiconBidAdapter.js

```diff
@@ -165,9 +165,9 @@ export const spec = {
         source: {
           tid: bidRequest.transactionId
         },
-        tmax: config.getConfig('TTL') || 1000,
+        tmax: bidderRequest.timeout,
         imp: [{
-          exp: 300,
+          exp: config.getConfig('s2sConfig.defaultTtl'),
```

Comment:
is there always an `s2sConfig` (with `defaultTtl`) when video is used?  If not, I guess we want to send `undefined`?